stylesheets/Components/modal.css

```diff
@@ -0,0 +1,19 @@
+.modal-container{
+    max-width: 50%;
+    margin: auto;
+    background-color: var(--st-light-pink);
+    padding: 15px;
+}
+.modal-info, .modal-btn{
+    padding: 20px;
```

Comment:
use rem unit for better responive

Reply:
my root font-size is 20px,  and i don't want the padding to change as it was looking a bit odd at first when i used rems......it's less than 1 rem anyway...so i let it be

Reply:
thankyou for taking time out to review❤️